apps/scan/backend/src/store.ts

```diff
@@ -993,4 +1010,48 @@ export class Store {
       arePollWorkerCardPinsEnabled: result.arePollWorkerCardPinsEnabled === 1,
     };
   }
+
+  /**
+   * Gets the name of the directory that we're continuously exporting to, e.g.
+   * TEST__machine_SCAN-0001__2023-08-16_17-02-24. Returns undefined if not yet set.
+   */
+  getExportDirectoryName(): string | undefined {
+    const result = this.client.one(
+      'select export_directory_name as exportDirectoryName from export_directory_name'
+    ) as { exportDirectoryName: string } | undefined;
+    return result?.exportDirectoryName;
+  }
+
+  /**
+   * Stores the name of the directory that we'll be continuously exporting to
+   */
+  setExportDirectoryName(exportDirectoryName: string): void {
+    this.client.run('delete from export_directory_name');
+    this.client.run(
+      'insert into export_directory_name (export_directory_name) values (?)',
+      exportDirectoryName
+    );
+  }
+}
+
+/**
+ * Builds a ScannerStore object to be passed through to a CastVoteRecordExporter
+ */
+export function buildScannerStoreForCastVoteRecordExporter(
```

Comment:
What's the motivation for building this new store object vs just having the store implement the `ScannerStore` interface?

I could see it being a little confusing to have an extra wrapper layer that adds slight modifications to the logic.

Reply:
Yeah great callout - I was just trying to minimize modifications to the existing VxScan `Store` while allowing myself to use names that felt a bit clearer in new code, e.g. `.getBatches` instead of `.batchStatus`.

But upon closer inspection, the amount of work required to make the VxScan `Store` implement the `ScannerStore` interface is minimal, including renaming existing methods in the VxScan `Store`. Will modify and get rid of the wrapper, which I agree is a little confusing